include/cuda/std/detail/libcxx/include/cmath

```diff
@@ -308,6 +308,10 @@ long double    truncl(long double x);
 #include <math.h>
 #endif
 
+#if defined(__cuda_std__) && defined(_LIBCUDACXX_COMPILER_NVHPC)
```

Comment:
https://clang.godbolt.org/z/hMWf98ae1

There is some weirdness depending on the toolchain NVC++ uses. Not sure how we need to configure the NVHPC image to get it to emit some of these functions.

Reply:
I am strongly against us having to work around missing functionality. We should rather ask where those can be found.

Reply:
Fixed after consultation with David.